packages/ui/src/blocks/color-mode-select/color-mode-select.tsx

```diff
@@ -13,27 +13,33 @@ export function ColorModeSelect(): JSX.Element {
   const handleChange = (value: string) => {
     setTheme(value);
   };
+  const container = 'flex items-center justify-start gap-2';
   return (
     <ClientOnly>
       <Select
         value={theme || 'system'}
-        onChange={handleChange}
-        className="w-32"
-        placement="top"
+        onValueChange={handleChange}
+        className="w-36"
         aria-label="Mode selector"
       >
-        <Select.Option value="light">
-          <IconSun size={16} />
-          <span>Light</span>
-        </Select.Option>
-        <Select.Option value="dark">
-          <IconMoon size={16} />
-          <span>Dark</span>
-        </Select.Option>
-        <Select.Option value="system">
-          <IconSettings size={16} />
-          <span>System</span>
-        </Select.Option>
+        <Select.Item value="light">
+          <div className={container}>
+            <IconSun size={20} />
```

Comment:
The size `20` aligns with `IconCheck`
<img width="111" alt="image" src="https://user-images.githubusercontent.com/43892874/214513570-2213aff3-e918-48ed-83e2-6c1932f2d5ff.png">

Reply:
Also check it on browser since feather icon may have some extra paddings